integreat_compass/cms/views/index.py

```diff
@@ -1,9 +1,125 @@
+from ast import literal_eval
+
+from django.shortcuts import render
 from django.views.generic import TemplateView
 
+from ..constants import offer_group_types, offer_mode_types
+from ..models import Offer, Tag
+
 
 class IndexView(TemplateView):
```

Comment:
Also, while you're at it, in my opinion it would make sense to convert this to a Django [`ListView`](https://docs.djangoproject.com/en/4.2/ref/class-based-views/generic-display/#listview) to make use of a bit more inbuilt functionality...

Reply:
This should be applied now :)